src/screens/ImportWalletQRCodeScreen.tsx

```diff
@@ -118,6 +129,14 @@ class ImportWalletQRCodeScreen extends React.Component<Props, State> {
       // make sure it re-renders
       await sleep(100);
       try {
+        const wallet = new HDSegwitP2SHArWallet();
```

Comment:
so have do we add Air wallet in this flow? how can we import legacy wallet if we are always navigating for scanning no matter what type of wallet we want import?